### PR TITLE
mvp implementation of importing web components

### PR DESCRIPTION
**Pull request by kenkouot**
This is an initial integration of the new style-guide top-bar. It's only showing up on the discussions landing page for now, as it's a WIP and so is discussions. We'll continue to iterated on this and replace hard-coded strings.

### CODE REVIEW

**Comment by lizlux** [comment]
@garthwebb @mklucsarits could you give this PR a look?

File: bower.json [hunk 1]
```diff
@@ -17,9 +17,11 @@
     "weppy": "Wikia/weppy#0.9.2",
     "ember-performance-sender": "1.1.1",
     "vignette": "wikia/vignette-js#2.1.1",
-    "wikia-style-guide": "Wikia/style-guide#1.0.2",
+    "wikia-style-guide": "Wikia/style-guide#1.0.3",
     "numeral": "~1.5.3",
-    "cropper": "~0.11.0"
+    "cropper": "~0.11.0",
+    "polymer": "Polymer/polymer#~1.1.0",
+    "webcomponentsjs": "~0.7.11"
```
Comment (mklucsarits) [inline bower.json]: What is the relationship between polymer and webcomponentsjs? I thought webcomponentsjs was a polyfill that replaced polymer, making polymer not needed. Why are both needed?

File: gulp/tasks/build-vendor.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ var gulp = require('gulp'),
 
 gulp.task('build-vendor', ['vendor'], function () {
 	return piper(
-		gulp.src(['www/front/vendor/script.js/dist/script.js',
+		gulp.src([
+			'www/front/vendor/webcomponentsjs/webcomponents-lite.js',
+			'www/front/vendor/script.js/dist/script.js',
```
Comment (mklucsarits) [inline gulp/tasks/build-vendor.js]: This is a really generic name for a library. What does it do?

File: bower.json (hunk 1 above)
Comment (garthwebb) [inline bower.json]: Looks like polymer is based on webcomponents? (http://webcomponents.org/, scroll down to 'libraries')

File: gulp/tasks/build-vendor.js (hunk 2 above)
Comment (lizlux) [inline gulp/tasks/build-vendor.js]: script loading. Not sure what it's being used for here. @kenkouot?

File: bower.json (hunk 1 above)
Comment (lizlux) [inline bower.json]: @garthwebb is correct, we need webcomponents.js for polymer.

**Pull request merged**